lib/bundler/cli/config.rb

```diff
@@ -21,24 +21,28 @@ def run
         return
       end
 
-      unless valid_scope?(scope)
-        Bundler.ui.error "Invalid scope --#{scope} given. Please use --local or --global."
-        exit 1
-      end
+      if options[:parseable]
+        Bundler.ui.info(Bundler.settings[name])
```

Comment:
I think probably just return here instead of putting everything inside an `else` block.